### PR TITLE
Update vision examples

### LINKED ISSUE #47: Update examples for Camera Server & NIVision
Update Eclipse Plugin examples that use Camera Server and NI Vision after new Camera Server and OpenCV are added and NI Vision is removed.

NI Vision dependent examples (old game examples) should probably move to NIVision repo and installer rather than being ported.

### PR DESCRIPTION
**Pull request by AustinShalit**
closes #47 
solves https://github.com/wpilibsuite/EclipsePlugins/issues/75#issuecomment-269705991

Opened PR https://github.com/wpilibsuite/nivision/pull/6

### CODE REVIEW

File: edu.wpi.first.wpilib.plugins.java/resources/templates/examples/AxisCameraSample/src/org/usfirst/frc/team190/robot/Robot.java [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
+				// Tell the CvSink to grab a frame from the camera and put it
+				// in the source mat
+				cvSink.grabFrame(mat);
+				// Put a rectangle on the image
+				Imgproc.rectangle(mat, new Point(100, 100), new Point(400, 400),
+						new Scalar(255, 255, 255), 5);
+				// Give the output stream a new image to display
+				outputStream.putFrame(mat);
+			}
+		}).start();
```
Comment (ThadHouse) [inline edu.wpi.first.wpilib.plugins.java/resources/templates/examples/AxisCameraSample/src/org/usfirst/frc/team190/robot/Robot.java]: We should either set this to be a domain thread, or use the new VisionThread

**Review by bradamiller — COMMENTED** [review]
It looks like the 2015 examples are showing up in the list of examples, but the files are missing. Does this commit include the XML files? Then the entire sample corresponding sample folder has to be deleted.

**Comment by AustinShalit** [comment]
Totally forgot about the xml file. Will update.

**Comment by PeterJohnson** [comment]
This should check the return value from GrabFrame() to see if an error occurred (return value of zero) and if so, report GetError() somewhere useful.  You can print it, but perhaps even more useful in the long run would be to call NotifyError() on the output side (to pass it along to the mjpg server)

**Review by PeterJohnson — APPROVED** [review]

**Pull request merged**